2d/linear_waves_flat/TimeSeriesPlot_v5.py

```diff
@@ -0,0 +1,241 @@
+#from scipy import *
+from pylab import *
+import numpy 
+import collections as cll
+from tank import *
+#from math import pi
+
+# Read data
+Datasource=1 #or    0:Paraview gauges 1:Gauge functions 2:Vof integration 3:Zero crossing analysis
+Variable=2  #       0:y;              1:u;              2:v
+
+
+if Datasource!=3:
+
+ NumberOfCases=1
+ x_loc=[]
+ for i in range(0,int(L[0]/gauge_dx+1)): #+1 only if gauge_dx is an exact 
+   x_loc.append(gauge_dx*i)
+
+ NumLocations=int(len(x_loc))
+ NumCol=3*NumLocations +1
+ y_loc=-0.5 #y=0 in the mean free surface
```

Comment:
@ezve and @adimako , it looks to me like this is the problem. The point gauges are located at `y=0.5` and `y=-0.5` is underneath the tank.